apps/clinic/src/modules/doctors/dtos/doctor-query.dto.ts

```diff
@@ -1,6 +1,19 @@
 import { ApiProperty } from '@nestjs/swagger';
 
+// Find way how to import it from SelectQueryBuilder orderBys.direction
+export type QueryBuilderSortDirection = 'ASC' | 'DESC';
+
 export class DoctorQueryDto {
   @ApiProperty({ required: false })
   search: string;
+
+  @ApiProperty({ required: false })
+  offset: number;
+  @ApiProperty({ required: false })
+  limit: number;
+
+  @ApiProperty({ required: false })
+  sortBy: string; // groupBy
+  @ApiProperty({ required: false })
+  sortDirection: QueryBuilderSortDirection; // orderBy
 }
```

Comment:
you should add type checking and enum validation, sortBy can be an enum too, just like sortDirection, but nothing is being validated right now, it seems

you can also add the enum in the ApiProperty, so the documentaion would be more comprehensive